tests/node-e2e/remote_dns/test_remote_dns_resolving_works.mjs

```diff
@@ -0,0 +1,27 @@
+import { createServer } from "net";
+
+console.log(">> test_remote_remote_dns_resolving_works");
+
+const server = createServer();
+server.on("connection", handleConnection);
```

Comment:
not sure how this tests it?

Reply:
Yeah, this was just my "does enabling the feature causes a crash?" test.